docs/integrations/bedrock.md

```diff
@@ -59,10 +91,11 @@ import instructor
 from pydantic import BaseModel
 import asyncio
 
-async_client = instructor.from_provider(
-    "bedrock/anthropic.claude-3-sonnet-20240229-v1:0",
-    async_client=True,
-)
+# Initialize the Bedrock client
+bedrock_client = boto3.client('bedrock-runtime')
+
+# Enable instructor patches for async Bedrock client
+async_client = instructor.from_bedrock(bedrock_client, _async=True)
```

Comment:
can we update this to `async_client=True` and make it backwards compartible with `_async` in the from bedrock api to standardize it a bit